core/server/guilds.js

```diff
@@ -117,7 +117,7 @@ Meteor.methods({
     if (!canEditGuild(Meteor.user(), guild)) throw new Meteor.Error('not-authorized', `Missing permissions to edit team members`);
 
     let users = Meteor.users.find({ _id: { $in: userIds }, guildId });
-    if (users.length !== userIds.length) throw new Meteor.Error('user-invalid', 'Given user is not in the team');
+    if (users.count() !== userIds.length) throw new Meteor.Error('user-invalid', 'Given user is not in the team');
```

Comment:
It won't work, you compare the number of users in the guild to the number of users you want to delete.

The basic condition was there to check that the user to be deleted is in the guild. This part is is now useless, but you have to be careful with the conditions bellow.

You can:
- Remove the lines 119 & 220
- Specify the guild in the update condition 
- In the condition line 125 make sure that the users are not in a guild anymore by checking that guildId does not exist on the users

```
// line 122
Meteor.users.update({ _id: { $in: userIds }, guildId }, { $unset: { guildId: 1 } });

// line 125
const usersUpdated = Meteor.users.find({ _id: { $in: userIds }, guildId: { exists: false } }).fetch();
```